### ISSUE TITLE
iOS 14 - Play button disappears resulting in no way to start video playback.

### ISSUE THREAD

**Issue by mdcone** [TheWidlarzGroup/react-native-video#2144]
# Bug

Once the video preview has loaded, the play button appears for a second to then disappear on iOS 14 beta simulators and devices, resulting in a video preview that cannot be played. 

## Platform
Which player are you experiencing the problem on:
* iOS (all iOS 14 betas)

## Environment info

React native info output:

```bash
System:
    OS: macOS 10.15.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 2.98 GB / 32.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.18.2 - ~/.nvm/versions/node/v12.18.2/bin/node
    Yarn: Not Found
    npm: 6.14.5 - ~/.nvm/versions/node/v12.18.2/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 13.7, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.0 AI-193.6911.18.40.6626763
    Xcode: 11.7/11E801a - /usr/bin/xcodebuild
  Languages:
    Java: 14.0.2 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: ^16.11.0 => 16.11.0 
    react-native: ^0.62.2 => 0.62.2 
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: 5.1.0-alpha8 though this happened on non-alpha versions of the library

## Steps To Reproduce

1. Run application on iOS 14 simulator or device.
2. Does not have to be built using XCode 12 (building with XCode 11.x and installing on an iOS 14 device via TestFlight or App Store yields same result)
...

## Expected behaviour

1. Play button stays and can be interacted with.

**Comment by eelmoni** [comment]
Hi there, by chance, Does anyone have an update on this?

**Comment by mdcone** [comment]
To get around this, I had to add a touchable overlay with a play icon which onPress flipped the paused prop on the Video component to true and hid the play button. Once the video playback started the normal controls were displayed and functioned properly.

**Comment by mdcone** [comment]
Update: Looks like it works just fine in 14.2 Beta 1.

**Comment by asalem1** [comment]
@mdcone any chance you've got a snippet of that play button component overlay? I can't get the icon to display as an overlay to the video, not sure where I'm going wrong

**Comment by Lemaro86** [comment]
> To get around this, I had to add a touchable overlay with a play icon which onPress flipped the paused prop on the Video component to true and hid the play button. Once the video playback started the normal controls were displayed and functioned properly.

Can you write example please with play method?

**Comment by Lemaro86** [comment]
Does anyone know how to fix it?
If you want to play video without extension you can use webview.

**Comment by asalem1** [comment]
For anyone that's searching for an implemenation that worked for me, I set the controls to false and toggled the pause/play state using a boolean flag that was toggling using a hook:

```
  <TouchableOpacity onPress={() => setVideoIsPaused(!videoIsPaused)}>
      <Video
        source={{
          uri: source
        }}
        style={{
          height: 400,
          position: 'relative',
          bottom: 25,
          width: Dimensions.get('window').width,
          zIndex: 0,
          backgroundColor: 'black',
        }}
        resizeMode="cover"
        controls={false}
        paused={videoIsPaused}
        repeat={true}
      />
      <Icon
        type="font-awesome-5"
        name="play"
        size={65}
        style={{
          zIndex: 9999,
          display: videoIsPaused ? 'flex' : 'none',
        }}
        containerStyle={{
          position: 'absolute',
          height: 400,
          bottom: 25,
          left: 0,
          right: 0,
          flexDirection: 'row',
          alignItems: 'center',
          justifyContent: 'center',
          backgroundColor: videoIsPaused ? 'rgba(0,0,0, .5)' : 'transparent',
        }}
        color={'rgba(255, 255, 255, 0.5)'}
      />
    </TouchableOpacity>
```

**Comment by cem2ran** [comment]
Also experiencing this issue, used the same workaround.

**Comment by vrgimael** [comment]
I had other completely unrelated video issues on iOS 14 that were also resolved in 14.2 Beta as pointed out by @mdcone

**Comment by mdcone** [comment]
@Asalem1, sorry for the late response. Your solution looks great!

**Comment by VitaliiTsurkan** [comment]
Also have the same video issues on iOS 14 and also noticed that they were resolved in 14.2 Beta
